src/azure-cli/azure/cli/command_modules/keyvault/_completers.py

```diff
@@ -4,21 +4,14 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.decorators import Completer
-from azure.cli.core._profile import Profile
-
-
-def _get_token(cli_ctx, server, resource, scope):  # pylint: disable=unused-argument
-    return Profile(cli_ctx=cli_ctx).get_login_credentials(resource)[0]._token_retriever()  # pylint: disable=protected-access
```

Comment:
This `get_token` never works because it doesn't comply with `KeyVaultAuthentication`'s `authorization_callback` signature.